pkg/cmd/dictionary/entries/browse/browse.go

```diff
@@ -0,0 +1,161 @@
+
+	PrintFlags *cmdutil.PrintFlags
+}
+
+// DictionaryEntry can be plural, compound or stopword entry.
+type DictionaryEntry struct {
+	Type          shared.EntryType
+	Word          string   `json:"word,omitempty"`
+	Words         []string `json:"words,omitempty"`
+	Decomposition string   `json:"decomposition,omitempty"`
+	ObjectID      string
+	Language      string
+}
+
+// NewBrowseCmd creates and returns a browse command for dictionnaries' entries.
+func NewBrowseCmd(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Command {
+	cs := f.IOStreams.ColorScheme()
+
+	opts := &BrowseOptions{
+		IO:           f.IOStreams,
+		Config:       f.Config,
+		SearchClient: f.SearchClient,
+		PrintFlags:   cmdutil.NewPrintFlags().WithDefaultOutput("json"),
+	}
+	cmd := &cobra.Command{
+		Use:       "browse {<dictionary>... | --all} [--include-defaults]",
+		Args:      cobra.OnlyValidArgs,
+		ValidArgs: shared.DictionaryNames(),
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return shared.DictionaryNames(), cobra.ShellCompDirectiveNoFileComp
+		},
+		Short: "Browse dictionary entries",
+		Long: heredoc.Docf(`
+			This command retrieves all entries from the specified %s dictionnaries.
+		`, cs.Bold("custom")),
+		Example: heredoc.Doc(`
+			# Retrieve all entries from the "stopword" dictionary (doesn't include default stopwords)
+			$ algolia dictionary entries browse stopword
+
+			# Retrieve all entries from the "stopword" and "plural" dictionnaries
+			$ algolia dictionary entries browse stopword plural
+
+			# Retrieve all entries from all dictionnaries
+			$ algolia dictionary entries browse --all
+
+			# Retrieve all entries from the "stopword" dictionnaries (including default stopwords)
+			$ algolia dictionary entries browse stopword --include-defaults
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.All && len(args) > 0 || !opts.All && len(args) == 0 {
+				return cmdutil.FlagErrorf("Either specify dictionaries' names or use --all to browse all dictionaries")
+			}
+
+			if opts.All {
+				opts.Dictionnaries = []search.DictionaryName{search.Stopwords, search.Plurals, search.Compounds}
+			} else {
+				opts.Dictionnaries = make([]search.DictionaryName, len(args))
+				for i, dictionnary := range args {
+					opts.Dictionnaries[i] = search.DictionaryName(dictionnary)
+				}
+			}
+
+			return runBrowseCmd(opts)
+		},
+	}
+
+	cmd.Flags().BoolVarP(&opts.All, "all", "a", false, "browse all dictionnaries")
+	cmd.Flags().BoolVarP(&opts.IncludeDefaultStopwords, "include-defaults", "d", false, "browse dictionaries and include default stopwords")
+
+	opts.PrintFlags.AddFlags(cmd)
+
+	return cmd
+}
+
+// runBrowseCmd executes the browse command
+func runBrowseCmd(opts *BrowseOptions) error {
+	cs := opts.IO.ColorScheme()
+	client, err := opts.SearchClient()
+	if err != nil {
+		return err
+	}
+
+	dictionaries := opts.Dictionnaries
```

Comment:
I don't see any added value for this declaration, why not directly use `opts.Dictionnaries` below?